src/registry.js

```diff
@@ -117,8 +117,7 @@ class CustomAttributeRegistry {
       }
     }
     // Attribute was removed
-    else if(newVal == null && !!inst.value) {
-      inst.value = newVal;
```

Comment:
The reasoning here is that `this.value`  will show the last value that the Custom Attribute had while being disconnected.

So it's more of like `null` doesn't exist in the eyes of a Custom Attribute, instead it should just get disconnected. The `null` is a value in the eyes of an element (f.e. Custom Element), so that it knows an attribute is not connected (the custom attributes that this is the case when `disconnectedCallback` is called).

In my use cases, I found it useful to know what the last value was during disconnected (and I can automatically infer the DOM's is `null` due to `disconnectedCallback`, though in practice I never need the `null` value, because I already know when my custom attribute is disconnected and can react to that).